src/main/java/org/kuali/kfs/module/ar/service/impl/ContractsGrantsInvoiceCreateDocumentServiceImpl.java

```diff
@@ -1081,8 +1082,6 @@ public List<ContractsGrantsInvoiceDetail> generateValuesForCategories(
                         invoiceDetailAccountObjectCodesForCategory) {
                     invDetail.setCumulativeExpenditures(invDetail.getCumulativeExpenditures()
                             .add(invoiceDetailAccountObjectCode.getCumulativeExpenditures()));
-                    invDetail.setInvoiceAmount(invDetail.getInvoiceAmount()
-                            .add(invoiceDetailAccountObjectCode.getCurrentExpenditures()));
```

Comment:
Should a comment be added to indicate that these two lines were removed by the FINP-10147 backport?